extensions/positron-python/src/client/common/process/pythonEnvironment.ts

```diff
@@ -173,6 +174,23 @@ export async function createCondaEnv(
     return new PythonEnvironment(interpreterPath, deps);
 }
 
+export async function createPixiEnv(
```

Comment:
trying things out: we are able to discover pixi environments 🎉 our mechanism for installing `ipykernel` fails, but if you have it already installed, everything seems to work as normal

![Screenshot 2024-07-02 at 2 25 15 PM](https://github.com/posit-dev/positron/assets/54685329/b740b367-aac3-4d48-a594-566a9571cb68)